### PR TITLE
Stats3

### PR DESCRIPTION
**Pull request by aheritier**
Testing @fcamblor work on #202 I found some problems in how we use the button groups (due to bootstrap upgrade) and the JS code wasn't updated.
(I also fixed a button size in another page)

⚠️ I also changed the order of the button. I always saw the - on the left and + on the right

I aligned the buttons to the right to be sure that they are always on the same verticale (whatever the note is)

Before:

Screenshot 2020-01-07 15 49 35

After:

Screenshot 2020-01-07 16 01 36

### CODE REVIEW

**Comment by fcamblor** [comment]
From a UI point of view :
- removing margin outside +/- buttons makes me feel user may misclick on "-" while wanting to click on "+" (particularly when you made +/- buttons smaller)
- I would enforce that +/- buttons have the same size (same width, `34px`)
- I would "surround" current score with -/+ buttons instead of putting those button on the right of the score

tldr; something like 
All_my_votes_-_Devoxx_France_2020

WDYT ?

**Comment by aheritier** [comment]
I buy it @fcamblor do you already have a PR ?

**Comment by aheritier** [comment]
(if yes - take care of the javascript which is used when you click and displays the refresh button instead of - / + buttons)

**Comment by fcamblor** [comment]
Wow ... didn't saw this refresh thing, that's weird (didn't implemented it that way at the beginning)

What is the point of this "sync" button instead of the original impl of sending directly the AJAX call when we click the +/- because I see some limitations here :
- We can only change our votes from +1/-1 at once, and need to click & sync twice if we want to apply -2/+2
- We can miss to click on the sync button
- We don't have a global sync button allowing to apply every "pending" sync at once
- When we change score, lines are not sorted anymore

I may understand that we don't want to call too much time the backend, but it lead to a very poor UX

I can try to build something keeping more or less the same backend call limitations restrictions by :
- Keeping +/- buttons displayed (but adding the refresh button when new score differs from initial score)
- Providing a global "sync all" button making the exact same action than clicking every visible "sync" buttons
- Everytime score is updated, trigger a table sort (by score)

WDYT ?

**Comment by aheritier** [comment]
I discovered the JS yesterday when I reviewed your PR and thus mine to try to fixup / cleanup a bit the HTML code and JS code for the new bootstrap.
I don't know why cc @nicmarti 
But yes your proposal could be great.

**Comment by fcamblor** [comment]
I can remember the JS part was (at the beginning) from me

But I guess it has been simplified (spoiler : it was overcomplex :-)) over time as I was completely building the whole table with JS and thus, was needing a JSON-ified view of the data (proposals, score per proposal and so on)

I think I can keep doing things I mentioned earlier without having to complexify too much things (the trickiest part will be to sort the table, but I think I should be able to mount every `<tr>` in memory, sort it, then recreate the corresponding DOM from it)

**Comment by nicmarti** [comment]
Can I merge the PR ?

**Comment by aheritier** [comment]
No, let's close this one @nicmarti 
We'll see if @fcamblor can do better

**Comment by fcamblor** [comment]
Yes I don't think it will take me too much time to implement, will try by this evening :-)
